src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './ProductList.scss';
 
 function ProductList() {
-  return <p>Hi</p>;
+  const [productInfoList, setProductInfoList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productList.json', { method: 'GET' })
+      .then(response => response.json())
+      .then(result => setProductInfoList(result));
+  }, []);
+  return (
+    <div className="productListContainerWrap">
+      <button className="slideLeft">왼쪽</button>
+      <div className="productListContainer">
+        {productInfoList.map((productInfo, index) => (
+          <ul key={index}>
+            <li className="productImg">
+              <img src={productInfo.productImage} alt="product" />
+            </li>
+            <div className="productDetailWrap">
+              <li className="productEng">{productInfo.englishName}</li>
+              <li className="productKor">{productInfo.koreanName}</li>
+              <li className="priceTag">{productInfo.price}</li>
+              <li className="sizeTag">{productInfo.size}</li>
+            </div>
+          </ul>
+        ))}
```

Comment:
```suggestion
        {productInfoList.map(({id, productImage, englishName, koreanName, price, size}) => (
          <ul key={id}>
            <li className="productImg">
              <img src={productImage} alt="product" />
            </li>
            <div className="productDetailWrap">
              <li className="productEng">{englishName}</li>
              <li className="productKor">{koreanName}</li>
              <li className="priceTag">{price}</li>
              <li className="sizeTag">{size}</li>
            </div>
          </ul>
        ))}
```

객체 안에 여러 프로퍼티가 있을 때, 이와 같이 객체구조분해 할당을 해서 사용할 수 있습니다!
그리고 key값에 index보다는 고유한 값으로 존재하는 id 값을 사용하는 것이 좋습니다!